{"uid":"43c54bd39681b655","name":"\"before all\" hook for \"Validar exclusão de uma lista com sucesso\"","fullName":"Validar o enpoint DELETE /list/{list_id}.\"before all\" hook for \"Validar exclusão de uma lista com sucesso\"","historyId":"dc145f0c6d5bd2679e344001ac30c784","time":{"start":1689965013956,"stop":1689965013956,"duration":0},"description":"","descriptionHtml":"","status":"broken","statusMessage":"`cy.request()` failed on:\n                \n                https://api.themoviedb.org/3/list\n                \n                The response we received from your web server was:\n                \n                  > 422: Unprocessable Entity\n                \n                This was considered a failure because the status code was not `2xx` or `3xx`.\n                \n                If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: POST\n                URL: https://api.themoviedb.org/3/list\n                Headers: {\n                  \"Connection\": \"keep-alive\",\n                  \"Authorization\": \"Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI5ODE4ZDY4Nzc4MDcwYTY5ZGZmZmFmNDk3ZDIwNmM2YiIsInN1YiI6IjVmZWNiNDNkMGYzNjU1MDA0MGFhNDQ2YSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.StSlwHbuW2FlD3BsS2Gm2A6eFANRUphotZJs39R2ipE\",\n                  \"user-agent\": \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/12.17.1 Chrome/106.0.5249.51 Electron/21.0.0 Safari/537.36\",\n                  \"accept\": \"*/*\",\n                  \"accept-encoding\": \"gzip, deflate\",\n                  \"content-type\": \"application/json\",\n                  \"content-length\": 130\n                }\n                Body: {\"name\":\"Teste da automação\",\"description\":\"Lista criada para que os testes automatizados sejam executados.\",\"language\":\"pt-br\"}\n                \n                -----------------------------------------------------------\n                \n                The response we got was:\n                \n                Status: 422 - Unprocessable Entity\n                Headers: {\n                  \"content-type\": \"application/json\",\n                  \"content-length\": \"65\",\n                  \"connection\": \"keep-alive\",\n                  \"date\": \"Fri, 21 Jul 2023 18:43:33 GMT\",\n                  \"server\": \"openresty\",\n                  \"cache-control\": \"public, max-age=0\",\n                  \"x-cache\": \"Error from cloudfront\",\n                  \"via\": \"1.1 10bf5e2f3ff7eaefa287c49657b8a838.cloudfront.net (CloudFront)\",\n                  \"x-amz-cf-pop\": \"DFW57-P2\",\n                  \"alt-svc\": \"h3=\\\":443\\\"; ma=86400\",\n                  \"x-amz-cf-id\": \"CIsIGNOP7-z_lYDL07ptrbAy6x1u2O8-nA22jEAxveVNXgaCUMZs3Q==\",\n                  \"vary\": \"Origin\"\n                }\n                Body: {\n                  \"errors\": [\n                    \"Invalid description\"\n                  ],\n                  \"success\": false,\n                  \"list_id\": null\n                }\n                \n                \n                https://on.cypress.io/request\n                \n                Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validar o enpoint DELETE /l...`","statusTrace":"CypressError: `cy.request()` failed on:\n                \n                https://api.themoviedb.org/3/list\n                \n                The response we received from your web server was:\n                \n                  > 422: Unprocessable Entity\n                \n                This was considered a failure because the status code was not `2xx` or `3xx`.\n                \n                If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: POST\n                URL: https://api.themoviedb.org/3/list\n                Headers: {\n                  \"Connection\": \"keep-alive\",\n                  \"Authorization\": \"Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI5ODE4ZDY4Nzc4MDcwYTY5ZGZmZmFmNDk3ZDIwNmM2YiIsInN1YiI6IjVmZWNiNDNkMGYzNjU1MDA0MGFhNDQ2YSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.StSlwHbuW2FlD3BsS2Gm2A6eFANRUphotZJs39R2ipE\",\n                  \"user-agent\": \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/12.17.1 Chrome/106.0.5249.51 Electron/21.0.0 Safari/537.36\",\n                  \"accept\": \"*/*\",\n                  \"accept-encoding\": \"gzip, deflate\",\n                  \"content-type\": \"application/json\",\n                  \"content-length\": 130\n                }\n                Body: {\"name\":\"Teste da automação\",\"description\":\"Lista criada para que os testes automatizados sejam executados.\",\"language\":\"pt-br\"}\n                \n                -----------------------------------------------------------\n                \n                The response we got was:\n                \n                Status: 422 - Unprocessable Entity\n                Headers: {\n                  \"content-type\": \"application/json\",\n                  \"content-length\": \"65\",\n                  \"connection\": \"keep-alive\",\n                  \"date\": \"Fri, 21 Jul 2023 18:43:33 GMT\",\n                  \"server\": \"openresty\",\n                  \"cache-control\": \"public, max-age=0\",\n                  \"x-cache\": \"Error from cloudfront\",\n                  \"via\": \"1.1 10bf5e2f3ff7eaefa287c49657b8a838.cloudfront.net (CloudFront)\",\n                  \"x-amz-cf-pop\": \"DFW57-P2\",\n                  \"alt-svc\": \"h3=\\\":443\\\"; ma=86400\",\n                  \"x-amz-cf-id\": \"CIsIGNOP7-z_lYDL07ptrbAy6x1u2O8-nA22jEAxveVNXgaCUMZs3Q==\",\n                  \"vary\": \"Origin\"\n                }\n                Body: {\n                  \"errors\": [\n                    \"Invalid description\"\n                  ],\n                  \"success\": false,\n                  \"list_id\": null\n                }\n                \n                \n                https://on.cypress.io/request\n                \n                Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validar o enpoint DELETE /l...`\n                    at <unknown> (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:149364:80)\n                    at tryCatcher (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:18744:23)\n                    at Promise._settlePromiseFromHandler (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:16679:31)\n                    at Promise._settlePromise (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:16736:18)\n                    at Promise._settlePromise0 (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:16781:10)\n                    at Promise._settlePromises (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:16861:18)\n                    at _drainQueueStep (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:13451:12)\n                    at _drainQueue (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:13444:9)\n                    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:13460:5)\n                    at Async.drainQueues (https://api.themoviedb.org/__cypress/runner/cypress_runner.js:13330:14)\n                From Your Spec Code:\n                    at Context.eval (webpack:///./cypress/integration/api-tests/lists/delete_list.spec.js:5:7)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"Validar o enpoint DELETE /list/{list_id}"},{"name":"testClass","value":"Validar o enpoint DELETE /list/{list_id}"},{"name":"testMethod","value":"\"before all\" hook for \"Validar exclusão de uma lista com sucesso\""},{"name":"package","value":"Validar o enpoint DELETE /list/{list_id}"},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"b8f4329e7f0cf27a","reportUrl":"https://pauloliveiram.github.io/tmdb-cypress/2//#testresult/b8f4329e7f0cf27a","status":"broken","statusDetails":"`cy.request()` failed on:\n                \n                https://api.themoviedb.org/3/list\n                \n                The response we received from your web server was:\n                \n                  > 422: Unprocessable Entity\n                \n                This was considered a failure because the status code was not `2xx` or `3xx`.\n                \n                If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: POST\n                URL: https://api.themoviedb.org/3/list\n                Headers: {\n                  \"Connection\": \"keep-alive\",\n                  \"Authorization\": \"Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI5ODE4ZDY4Nzc4MDcwYTY5ZGZmZmFmNDk3ZDIwNmM2YiIsInN1YiI6IjVmZWNiNDNkMGYzNjU1MDA0MGFhNDQ2YSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.StSlwHbuW2FlD3BsS2Gm2A6eFANRUphotZJs39R2ipE\",\n                  \"user-agent\": \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/12.17.1 Chrome/106.0.5249.51 Electron/21.0.0 Safari/537.36\",\n                  \"accept\": \"*/*\",\n                  \"accept-encoding\": \"gzip, deflate\",\n                  \"content-type\": \"application/json\",\n                  \"content-length\": 130\n                }\n                Body: {\"name\":\"Teste da automação\",\"description\":\"Lista criada para que os testes automatizados sejam executados.\",\"language\":\"pt-br\"}\n                \n                -----------------------------------------------------------\n                \n                The response we got was:\n                \n                Status: 422 - Unprocessable Entity\n                Headers: {\n                  \"content-type\": \"application/json\",\n                  \"content-length\": \"65\",\n                  \"connection\": \"keep-alive\",\n                  \"date\": \"Fri, 21 Jul 2023 18:18:11 GMT\",\n                  \"server\": \"openresty\",\n                  \"cache-control\": \"public, max-age=0\",\n                  \"x-cache\": \"Error from cloudfront\",\n                  \"via\": \"1.1 a4cae74c829bc214e4183c38164a2c0a.cloudfront.net (CloudFront)\",\n                  \"x-amz-cf-pop\": \"IAD89-C3\",\n                  \"alt-svc\": \"h3=\\\":443\\\"; ma=86400\",\n                  \"x-amz-cf-id\": \"OIzJanJHGOUv2B8n32ncFY3pUpuoIH4LZtrY2xy1ttQ7IKDIoRgB4Q==\",\n                  \"vary\": \"Origin\"\n                }\n                Body: {\n                  \"errors\": [\n                    \"Invalid description\"\n                  ],\n                  \"success\": false,\n                  \"list_id\": null\n                }\n                \n                \n                https://on.cypress.io/request\n                \n                Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validar o enpoint DELETE /l...`","time":{"start":1689963491581,"stop":1689963491581,"duration":0}},{"uid":"667c5a365fcdc531","reportUrl":"https://pauloliveiram.github.io/tmdb-cypress/1//#testresult/667c5a365fcdc531","status":"broken","statusDetails":"`cy.request()` failed on:\n                \n                https://api.themoviedb.org/3/list\n                \n                The response we received from your web server was:\n                \n                  > 422: Unprocessable Entity\n                \n                This was considered a failure because the status code was not `2xx` or `3xx`.\n                \n                If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: POST\n                URL: https://api.themoviedb.org/3/list\n                Headers: {\n                  \"Connection\": \"keep-alive\",\n                  \"Authorization\": \"Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI5ODE4ZDY4Nzc4MDcwYTY5ZGZmZmFmNDk3ZDIwNmM2YiIsInN1YiI6IjVmZWNiNDNkMGYzNjU1MDA0MGFhNDQ2YSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.StSlwHbuW2FlD3BsS2Gm2A6eFANRUphotZJs39R2ipE\",\n                  \"user-agent\": \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/12.17.1 Chrome/106.0.5249.51 Electron/21.0.0 Safari/537.36\",\n                  \"accept\": \"*/*\",\n                  \"accept-encoding\": \"gzip, deflate\",\n                  \"content-type\": \"application/json\",\n                  \"content-length\": 130\n                }\n                Body: {\"name\":\"Teste da automação\",\"description\":\"Lista criada para que os testes automatizados sejam executados.\",\"language\":\"pt-br\"}\n                \n                -----------------------------------------------------------\n                \n                The response we got was:\n                \n                Status: 422 - Unprocessable Entity\n                Headers: {\n                  \"content-type\": \"application/json\",\n                  \"content-length\": \"65\",\n                  \"connection\": \"keep-alive\",\n                  \"date\": \"Fri, 21 Jul 2023 17:35:17 GMT\",\n                  \"server\": \"openresty\",\n                  \"cache-control\": \"public, max-age=0\",\n                  \"x-cache\": \"Error from cloudfront\",\n                  \"via\": \"1.1 2160425a36a5a0604048dda5b151b504.cloudfront.net (CloudFront)\",\n                  \"x-amz-cf-pop\": \"SFO53-C1\",\n                  \"alt-svc\": \"h3=\\\":443\\\"; ma=86400\",\n                  \"x-amz-cf-id\": \"iRBusJp5hLxR8FMXH9kketOu6OEzN8lohQgZW1ixUHJw651doUDweQ==\",\n                  \"vary\": \"Origin\"\n                }\n                Body: {\n                  \"errors\": [\n                    \"Invalid description\"\n                  ],\n                  \"success\": false,\n                  \"list_id\": null\n                }\n                \n                \n                https://on.cypress.io/request\n                \n                Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validar o enpoint DELETE /l...`","time":{"start":1689960917567,"stop":1689960917567,"duration":0}}]},"tags":[]},"source":"43c54bd39681b655.json","parameterValues":[]}